step/src/main/webapp/chat.jsp

```diff
@@ -37,6 +37,7 @@ limitations under the License.
     </head>
     <body>
         <nav>
+            <a id="profile-button" href="profile.jsp"> Profile </a>
```

Comment:
Can you put something like "My Profile" here? I think just "Profile" could get confusing while displaying other users